### PR TITLE
Refactored organelles with submeshes

### LINKED ISSUE #2797: Binding agent stays membrane colour after exiting membrane tab in editor
Steps:
- Go to the editor and place a binding agent
- Switch to the membrane tab and select a colour
- Switch back to the structure tab
- Notice how the binding agent is coloured the membrane colour while other organelles are not

### LINKED ISSUE #2798: Binding agent's membrane recolors seemingly randomly
**Describe the bug**

Sometimes the membrane of the BA recolors

**To Reproduce**
Not sure, but happens after dying and after exiting the editor.

**Screenshots**
image

### PR DESCRIPTION
**Pull request by FuJa0815**
**Brief Description of What This PR Does**

Refactored the way how the game handles organelles with submeshes (Nucleus, BA)

**Related Issues**

closes #2797
closes #2798
fixed a bug where the Golgi and ER textures never get used

**Progress Checklist**

Note: before starting this checklist the PR should be marked as non-draft.

- [ ] PR author has checked that this PR works as intended and doesn't
      break existing features:
      https://wiki.revolutionarygamesstudio.com/wiki/Testing_Checklist
      (this is important as to not waste the time of Thrive team
      members reviewing this PR)
- [x] Initial code review passed (this and further items should not be checked by the PR author)
- [ ] Functionality is confirmed working by another person (see above checklist link)
- [ ] Final code review is passed and code conforms to the 
      [styleguide](https://github.com/Revolutionary-Games/Thrive/blob/master/doc/style_guide.md).

Before merging all CI jobs should finish on this PR without errors, if
there are automatically detected style issues they should be fixed by
the PR author.

### CODE REVIEW

**Comment by Kasterisk** [comment]
Code and functionality seems good but the default BA membrane color is just white now, is it possible to repaint the texture to its green color?

**Comment by hhyyrylainen** [comment]
Also did the binding agent colour change when in binding mode, does that work still?

**Comment by FuJa0815** [comment]
> Code and functionality seems good but the default BA membrane color is just white now, is it possible to repaint the texture to its green color?

Done.



> Also did the binding agent colour change when in binding mode, does that work still?

No, same as the master. The BA itself does not change color, only the membrane.

**Comment by Kemikal1** [comment]
I think thats unintentionall, it should easily be fixable with an exception.

**Comment by RacerBG** [comment]
> I think thats unintentionall, it should easily be fixable with an if exception.

Probably it's not even related to this PR but that should not happen in my opinion.

**Review by Kasterisk — APPROVED** [review]
Looks fine.

File: src/microbe_stage/PlacedOrganelle.cs [hunk 1]
```diff
@@ -33,6 +33,8 @@ public class PlacedOrganelle : Spatial, IPositionedOrganelle, ISaveLoadedTracked
     [JsonProperty]
     private Dictionary<Compound, float> compoundsLeft;
 
+    private Spatial organelleSceneInstance;
```
Comment (hhyyrylainen) [inline src/microbe_stage/PlacedOrganelle.cs:36]: Seems like this not being saved doesn't cause issues.

**Review by hhyyrylainen — APPROVED** [review]
Old saves seem to also work fine with this.

**Pull request merged**